src/main/java/com/google/cloud/tools/managedcloudsdk/command/CommandCaller.java

```diff
@@ -69,7 +69,19 @@ public String call(
       }
```

Comment:
This still doesn't fix #814. `CommandExitException` above should also be modified to include the stdOut and stdErr as an exception message like `CommandExecutionException` does in this PR (if this is the way we want to surface the outputs). `CommandExitException` accepts stdOut and stdErr as an argument in the constructor, but they are not set as an exception message.

Reply:
Correct. The information is in the exception, but the client needs to choose to surface that.

Reply:
And this PR does not fix #814. The error in #814 is about `CommandExitException`, but this PR doesn't change anything about `CommandExitException`. I think we should let #814 open.

Reply:
Resolved. I've removed "fixes #814" from your initial PR comment.